recipe/meta.yaml

```diff
@@ -7,20 +8,21 @@ package:
 source:
   fn: traitlets-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/t/traitlets/traitlets-{{ version }}.tar.gz
-  md5: d0725f9818bd9e112a6531da26d433b8
+  sha256: {{ sha256 }}
 
 build:
   number: 0
-  script: pip install --no-deps .
+  script: python setup.py install
```

Comment:
>  is there a link to the discussion of removing pip from Python in conda-forge?

That is scattered in many places but maybe @jjhelmus, who is our python-feedstock hero, can answer that.

> anaconda-installed Python lacked pip in the beginning,

> I would like to make sure that conda-forge doesn't make the same mistake again.

:+1: to that. I am trying to be vocal on that but I am only one. Maybe we could write an enhancement proposal to explain the prons and cons and try to  move towards the easier `pip install --no-deps .`

We have a meeting today and I will bring this issue.